src/main/java/frc/robot/subsystems/swerve/SwerveMath.java

```diff
@@ -20,7 +23,8 @@ public static ChassisSpeeds discretize(ChassisSpeeds chassisSpeeds) {
         return ChassisSpeeds.discretize(chassisSpeeds, CycleTimeUtils.getCurrentCycleTime());
     }
 
-    public static ChassisSpeeds powersToSpeeds(double xPower, double yPower, double thetaPower, DriveSpeed driveSpeed, SwerveConstants constants) {
+    public static ChassisSpeeds powersToSpeeds(double xPower, double yPower, double thetaPower, DriveSpeed driveSpeed,
+            SwerveConstants constants) {
```

Comment:
return to one line (i know its the ctrl alt l)

Reply:
i hate u